### ISSUE TITLE
Please upgrade ember-cli-babel to ^7

### ISSUE THREAD

**Issue by wukongrita** [RSSchermer/ember-multiselect-checkboxes#55]
Hello @RSSchermer,

I've got this warning when creating new project with Emberjs 4+ :

```
WARNING: [DEPRECATION] [DEPRECATION] Usage of the Ember Global is deprecated. You should import the Ember module or the specific API instead.

See https://deprecations.emberjs.com/v3.x/#toc_ember-global for details.

Usages of the Ember Global may be caused by an outdated ember-cli-babel dependency. The following steps may help:

* Upgrade the following addons to the latest version:
  * ember-multiselect-checkboxes

### Important ###

In order to avoid repeatedly showing the same deprecation messages, no further deprecation messages will be shown for usages of the Ember Global until ember-cli-babel is upgraded to v7.26.6 or above.

To see all instances of this deprecation message, set the `EMBER_GLOBAL_DEPRECATIONS` environment variable to "all", e.g. `EMBER_GLOBAL_DEPRECATIONS=all ember test`.

### Details ###

Prior to v7.26.6, ember-cli-babel sometimes transpiled imports into the equivalent Ember Global API, potentially triggering this deprecation message indirectly, even when you did not observe these deprecated usages in your code.

The following outdated versions are found in your project:

* ember-cli-babel@6.18.0, currently used by:
  * ember-multiselect-checkboxes@0.11.1
    * Depends on ember-cli-babel@^6.6.0
  * ember-runtime-enumerable-includes-polyfill@2.1.0 (Dormant)
    * Depends on ember-cli-babel@^6.9.0
    * Added by ember-multiselect-checkboxes@0.11.1

Note: Addons marked as "Dormant" does not appear to have any JavaScript files. Therefore, even if they are using an old version ember-cli-babel, they are unlikely to be the culprit of this deprecation and can likely be ignored.
```

Here is my environment:
```
ember -v
ember-cli: 4.12.2
node: 18.19.0
os: linux x64
```

Seems need to upgrade the ember-cli-babel and is there any plan to upgrade this great add on for supporting Emberjs 4+ ?

**Comment by RSSchermer** [comment]
It's been a long time since I've used Ember myself, so this addon has fallen quite far behind the current Ember version. I just spent a few hours this afternoon upgrading to Ember 3.28 LTS, which is the most recent version of the Ember major version this addon was last updated for, and published this as version `0.12.0`; this does include an update to the Babel version to `^7.26.10`. 

Does upgrading your `ember-multiselect-checkboxes` version to `0.12.0` resolve your issue?

**Comment by wukongrita** [comment]
Thank you @RSSchermer for responding me with the update of the new release..
Btw, I still have some problems when running the app, like message shown below:

![image](https://github.com/RSSchermer/ember-multiselect-checkboxes/assets/25361005/eee95121-91bd-462b-bd0f-6a5e44203074)

Is there anything that I missed ?


Here is my route & template:

routes/application.js
```
...
  async model() {
    return await this.store.findAll('corelocale');
  }
...
```

templates/application.hbs
```
<Multiselect-checkboxes
  @options={{@model}}
  @labelProperty='name'
  @selection={{this.selectedUsers}}
/>
```

PS:
Seems it works fine if the source data is just a simple array (like the sample at the demo folder). But gives error if the source data is `json-api` format from `ember-data`.

Can you guide me to solve it?

Since when I'm using the ember < 4, showing data with multiselect-checkboxes from `json-api` works fine...

Thank you...

**Comment by wukongrita** [comment]
I've solved it, set the `EXTEND_PROTOTYPES` value to `true` in `environment.js`..

Thank @RSSchermer for this great addon...
